awx/main/scheduler/tasks.py

```diff
@@ -9,16 +9,6 @@
 logger = logging.getLogger('awx.main.scheduler')
 
 
-@task()
-def run_job_launch(job_id):
-    TaskManager().schedule()
-
-
-@task()
-def run_job_complete(job_id):
-    TaskManager().schedule()
-
-
 @task()
 def run_task_manager():
```

Comment:
I wonder if long-term we could just move this single task into `awx.main.tasks` and lose this file.  It's the only thing in here now, and that's a little weird.

Reply:
no objections here